src/index.html

```diff
@@ -1,15 +1,197 @@
+  <meta
+    name="viewport"
+    content="
+      width=device-width,
+      user-scalable=no,
+      initial-scale=1.0,
+      maximum-scale=1.0,
+      minimum-scale=1.0
+    "
+  >
+  <meta
+    http-equiv="X-UA-Compatible"
+    content="ie=edge"
+  >
   <title>HTML Form</title>
-  <link rel="stylesheet" href="./style.css">
+  <link
+    rel="stylesheet"
+    href="./style.css"
+  >
 </head>
+
 <body>
-  <h1>HTML Form</h1>
+
+  <form
+    name="Personal information"
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    autocomplete="off"
+  >
+    <fieldset class="fieldset">
+      <legend>Personal information</legend>
+        <p>
+          <label>Surname:
+              <input
+                name="surname"
+                type="text"
+              >
+          </label>
+        </p>
+        <p>
+          <label>Name:
+            <input
+              type="text"
+              name="name"
+            >
+          </label>
+        </p>
+        <p>
+          <label>How old are You?
+            <input
+              type="number"
+              value="12"
+              min="1"
+              max="100"
+              name="age"
+            >
+          </label>
+        </p>
+        <p>
+          <label>Full date of birth:
+            <input
+              type="date"
+              name="birthDate"
+            >
+          </label>
+        </p>
+        <p>
+          <label>I accept the term of the agreement
+            <input
+              type="checkbox"
+              name="agree"
+            >
+          </label>
+        </p>
+    </fieldset>
+
+    <fieldset class="fieldset">
+      <legend>Registration</legend>
+      <p>
+        <label>E-mail:
+          <input
+            maxlength="42"
+            required
+            type="email"
+            placeholder="email@example.com"
+            name="youMail"
+          >
+        </label>
+      </p>
+      <p>
+        <label>Password:
+          <input
+            minlength="5"
+            required type="password"
+            name="pass"
+          >
+        </label>
+      </p>
+    </fieldset>
+
+    <fieldset class="fieldset">
+      <legend>An interesting fact about you!</legend>
+      <p>
+        <label>Do you love cats?
+          <input
+            type="radio"
+            name="cats"
+            value="yes"
+          > Yes
+          <input
+            type="radio"
+            name="cats"
+            value="no"
+          > No
+        </label>
+      </p>
+      <p>
+        <label>What is your favorite color?
+          <input
+            type="color"
+            name="rainbow"
+          >
+        </label>
+      </p>
+      <p>
+        <label>What time do you go to bed?
+          <input
+            type="time"
+            step="2"
+            name="SleepTime"
+          >
+        </label>
+      </p>
+      <p>
+        <label>What are your favorite brand of cars?
```

Comment:
```suggestion
        <label>
          What are your favorite brand of cars?
```
do like this for text inside tags